src/components/modals/signupModals/signupCollaborator3.tsx

```diff
@@ -17,6 +19,13 @@ export default function SignupModalCollaborator3() {
       <Dialog.Description className="Text w-full text-center text-midnight">
         {`let's change this job market together!`}
       </Dialog.Description>
+      {referralPartner ? (
```

Comment:
Just use the && operator to render this conditionally, you don't need to use a ternary operator here